pvacseq/lib/vaccine_design.py

```diff
@@ -0,0 +1,390 @@
+#  Input would be a protein fasta
+# ie:
+# python vaccine_design.py test peptides.fa -o output.csv ann  H-2-Kb -l 8
+import shutil
+import sys
+import argparse
+import os
+from pathlib import Path
+root = str(Path(__file__).resolve().parents[1])
+sys.path.append(root)
+
+import pandas
+import networkx as nx
+import itertools
+from Bio import SeqIO
+import lib
+
+import time
+from simanneal import Annealer
+import random
+import math
+
+from lib.prediction_class import *
+
+
+def define_parser():
+
+    parser = argparse.ArgumentParser('pvacseq vaccine_design')
+    parser.add_argument(
+        "run_name",
+        help="The name of the run being processed." +
+             " This will be used as a prefix for output files."
+    )
+    parser.add_argument('input_file', type=argparse.FileType('r'),
+                        help="Path to input FASTA file")
+    parser.add_argument('method',
+                        choices=PredictionClass.iedb_prediction_methods(),
+                        help="The iedb analysis method to use")
+    parser.add_argument('allele',
+                        help="Allele for which to make prediction")
+    parser.add_argument('-o', "--outdir", help="Output directory")
+    parser.add_argument("-k", "--keep-tmp", default=False,
+                        help="Option to store tmp files. " +
+                             "Default: False")
+    parser.add_argument(
+        "-l", "--epitope-length",
+        type=lambda s: [int(epl) for epl in s.split(',')],
+        help="Length of subpeptides (neoepitopes) to predict. " +
+             "Multiple epitope lengths can be specified " +
+             "using a comma-separated list. " +
+             "Typical epitope lengths vary between 8-11. " +
+             "Required for Class I prediction algorithms",
+    )
+    parser.add_argument("-c", "--cutoff", type=int,
+                        default=500,
+                        help="Optional ic50 cutoff value." +
+                             " Junctional neoepitopes with IC50 values below this " +
+                             " value will be excluded. Default: 500")
+    parser.add_argument(
+        "-r", "--iedb-retries", type=int,
+        default=5,
+        help="Number of retries when making requests to the IEDB RESTful web interface. " +
+             " Must be less than or equal to 100." +
+             "Default: 5"
+    )
+     parser.add_argument(
```

Comment:
I think your alignment is off by a single space here (just kinda eyeballing it from GH).  Travis is failing on this line due to an unexpected indentation.